acceptance/current/.kitchen.yml

```diff
@@ -1,9 +1,9 @@
 driver:
   name: vagrant
-  forward_agent: yes
```

Comment:
I thought that was just needed for vbox issues for some reason.  We can put it back.